package.json

```diff
@@ -33,7 +33,7 @@
     "handlebars": "^4.0.5",
     "mkdirp": "^0.5.1",
     "moment": "^2.14.1",
-    "typescript": "^2.1.0-dev.20160910",
+    "typescript": "^2.0.3",
```

Comment:
Why are you insisting on typescript@next? When I checked out your project I realized that it is quite unstable if not using the exact version you specified (typescript and typescript-formatter). Did you have similar issues or do you just have one version of your project installed which builds ever since?